new-log-viewer/src/services/LogFileManager/index.ts

```
@@ -24,20 +26,51 @@
 
+const SEARCH_CHUNK_SIZE = 10000;
+
+/**
+ * Loads a file from a given source.
+ *
+ * @param fileSrc The source of the file to load. This can be a string representing a URL, or a File
+ * object.
+ * @return A promise that resolves with an object containing the file name and file data.
+ * @throws {Error} If the file source type is not supported.
+ */
+const loadFile = async (fileSrc: FileSrcType)
```

Biome

[error] Shouldn't redeclare 'loadFile'. Consider to delete it or rename it.

'loadFile' is defined here:


(lint/suspicious/noRedeclare)